nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -279,6 +279,11 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 - The `networking.wireguard` module now can set the mtu on interfaces and tag its packets with an fwmark.
 
+- The option `overrideStrategy` was added to systemd unit definition to allow systemd template instance definition in NixOS.
+  If set to `asDropin` the unit definition is stored as drop-in file instead of a unit file.
+  So you are able to specify e.g. following service `systemd-nspawn@machine` as an extention to the service `systemd-nspawn@`.
+  See also example in fixed issue (NixOS's systemd abstraction doesn't work with systemd template units)[https://github.com/NixOS/nixpkgs/issues/80933].
```

Comment:
```suggestion
- The option `overrideStrategy` was added to the different systemd unit options (`systemd.services.<name>`, `systemd.sockets.<name>`, …) to allow enforcing the creation of a dropin file, rather than the main unit file, by setting it to `asDropin`.
- This is useful in cases where the existence of the main unit file is not known to Nix at evaluation time, for example when the main unit file is provided by adding a package to `systemd.packages`.
- See the fix proposed in [NixOS's systemd abstraction doesn't work with systemd template units](https://github.com/NixOS/nixpkgs/issues/135557#issuecomment-1295392470) for an example.